Demo/obj/Debug/netcoreapp3.0/Demo.csproj.FileListAbsolute.txt

```diff
@@ -82,3 +82,45 @@ C:\Users\Yas\source\repos\Demo\Demo\obj\Debug\netcoreapp3.0\Demo.Views.pdb
 C:\Users\Yas\source\repos\Demo\Demo\obj\Debug\netcoreapp3.0\Demo.csproj.CopyComplete
```

Comment:
Still see debug files

Reply:
This File does not exist in our future commit